.changeset/purple-ideas-switch.md

```diff
@@ -2,11 +2,16 @@
 "@fluidframework/tree": minor
```

Comment:
tree is reexported via fluid-framework, so this impacts users of the fluid-framework package as well:
```suggestion
"@fluidframework/tree": minor
"fluid-framework": minor
```